.chloggen/redis-use-configopaque.yaml

```diff
@@ -0,0 +1,20 @@
+# Use this changelog template to create an entry for release notes.
+# If your change doesn't affect end users, such as a test fix or a tooling change,
+# you should instead start your pull request title with [chore] or use the "Skip Changelog" label.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
```

Comment:
I think this maybe belongs to enhancement? Since this is not breaking the way of the usage

Reply:
Although its not a breaking change for end users, it's a breaking changes to users of the code as a library. There is some discussion on whether this should be included in the changelog [here](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/24014).